storage/replica.go

```diff
@@ -862,14 +865,22 @@ func (r *Replica) proposeRaftCommand(ctx context.Context, ba roachpb.BatchReques
 	}
 	cmdID := ba.GetOrCreateCmdID(r.store.Clock().PhysicalNow())
 	idKey := makeCmdIDKey(cmdID)
-	r.Lock()
-	r.pendingCmds[idKey] = pendingCmd
-	r.Unlock()
+
 	var errChan <-chan error
-	if r.proposeRaftCommandFn != nil {
-		errChan = r.proposeRaftCommandFn(idKey, raftCmd)
+	if r.quiesced {
```

Comment:
Clearly not awake yet. We don't have many tests that actually `Quiesce` (and the one I wrote is synchronized so that the race can't happen).